ci_framework/roles/artifacts/tasks/cluster_info.yml

```diff
@@ -46,12 +46,18 @@
         # prefixed with "pod/" so that the redirect places the logs
         # in the correct pod directory. That is why we do not cd after
         # creating the directory.
+        search_word="controller-manager"
```

Comment:
maybe it would be better to do double loop - as all_pods is generated just above by listing pods in namespaces?

e.g.:
  for ns in openstack-operators openstack; do
     for pod in $(oc get pods ... -n $ns); do
       ....
     done
  done
  # or alternatively 'for ns in $(oc get namespaces -o name)' though it prefixes them with "namespace/" too, and not sure if we would want really all openshift ones recorded here)